ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Ledger/Query.hs

```diff
@@ -239,10 +239,10 @@ instance ShelleyCompatible proto era => QueryLedger (ShelleyBlock proto era) whe
           getProposedPPUpdates st
         GetStakeDistribution ->
           SL.poolsByTotalStakeFraction globals st
-        GetUTxOByAddress addrs ->
-          SL.getFilteredUTxO st addrs
-        GetUTxOWhole -> -- FIXME:  if we never match this case then we should probaly clarify this to avoid confusions when reading this part of the code.
-          SL.getUTxO st
+        GetUTxOByAddress{} ->
```

Comment:
Does this impact the cli?

Reply:
The CLI will be impacted because of other things (now this query is quite expensive) but we are still working on this. This error call just indicates that this path should be unreachable, as a GetUTxOByAddress query should be handled by the `handleWholeQuery` handler instead.